SplunkOtel.podspec

```diff
@@ -0,0 +1,33 @@
+#
+# NOTE: Be sure to run
+#
+# `pod lib lint SplunkOtel.podspec`
+#
+#  to ensure this is a valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'SplunkOtel'  
+  s.version          = '0.1.0'  
```

Comment:
Possibly write some build automation to ensure this lines up with `SplunkRumVersionString` in `SplunkRum.swift`.  One idea off the top of my head is that in `fullbuild.sh` you could grep it out of the two locations (here and SplunkRum.swift) and fail the build if they don't match.